Fix order customer comment and product out of stock

jolelievre opened the pull request:
<!-----------------------------------------------------------------------------
Thank you for contributing to the PrestaShop project! 
Please take the time to edit the "Answers" rows below with the necessary information.
Check out our contribution guidelines to find out how to complete it:
https://devdocs.prestashop.com/1.7/contribute/contribution-guidelines/#pull-requests
------------------------------------------------------------------------------>

| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Some templates had been fixed directly in the core but not in the sources from this repo<br>Also update composer dependencies so that PHP 7.3 can be used to generate the mail theme
| Type?         | improvement
| BC breaks?    | yes
| Deprecations? | no
| Fixed ticket? | Fixes https://github.com/PrestaShop/PrestaShop/pull/22028
| How to test?  | You can regenerate the modern theme with the appropriate wordings (already tested by @jolelievre )

<!-- Click the form's "Preview" button to make sure the table is functional in GitHub. Thank you! -->

eternoendless commented on mails/themes/modern_mjml/core/productoutofstock.mjml.twig:
```diff
@@ -37,7 +37,7 @@
   <mj-column background-color="#fefefe" border="1px solid #DFDFDF" padding-top="10px" padding-bottom="10px">
     <mj-text>
-      {{ 'There are now less than [1]{last_qty}[/1] items in stock.'|trans({'[1]': '<span class="label">', '[/1]': '</span>'}, 'Emails.Body', locale)|raw }}
+      {{ 'There are now less than [1]{last_qty}[/1] of [1]{product}[/1] in stock.'|trans({'[1]': '<span class="label">', '[/1]': '</span>'}, 'Emails.Body', locale)|raw }}
```
```suggestion
      {{ 'There are now less than [1]{last_qty}[/1] items of [1]{product}[/1] in stock.'|trans({'[1]': '<span class="label">', '[/1]': '</span>'}, 'Emails.Body', locale)|raw }}
```

jolelievre replied:
I'm fine with it but it will add yet another wording in the catalog. `There are now less than [1]{last_qty}[/1] of [1]{product}[/1] in stock.` is present in the 177 catalog for now. Should we add this new wording for the patch version then?

jolelievre replied:
@Julievrz what do you think?

Julievrz replied:
Thanks 👍

eternoendless commented:
@jolelievre Would you mind pushing your last commit in a separate PR?

PierreRambaud reviewed (dismissed):
Could you please separate commits?

jolelievre commented:
You mean update the dependency in a separate PR?

PierreRambaud commented:
> You mean update the dependency in a separate PR?

Yep :)

jolelievre commented:
Ok @PierreRambaud I split the update into this PR https://github.com/PrestaShop/mjml-theme-converter/pull/10

And @Julievrz I made the modification to use the existing wording, so I added the Product line so that the merchant can still identify which product is concerned To answer your question two templates exist one is used by the core and another one by the module `ps_emailalert` I guess this one has been integrated in the core directly later 🤷‍♂️

okom3pom commented:
@jolelievre my last PR https://github.com/PrestaShop/mjml-theme-converter/pull/8 remove delete 

`{{ 'Hi {firstname} {lastname},'|trans({}, 'Emails.Body', locale) }}`

Because it's an email for the shop not for the customer

okom3pom commented:
Here it's also an email for the shop : 

https://github.com/PrestaShop/mjml-theme-converter/blob/master/mails/themes/modern_mjml/core/productoutofstock.mjml.twig#L12

Julievrz commented:
The wording looks good to me, but I could not find the label. ✔️

atomiix commented on mails/themes/modern_mjml/core/order_customer_comment.mjml.twig:
```diff
@@ -10,7 +10,7 @@
     </mj-raw>
     <mj-text padding-top="0" padding-bottom="20px" font-weight="600" font-size="20px">
-      {{ 'Hi,'|trans({}, 'Emails.Body', locale) }}
+      {{ 'Hi {firstname} {lastname},'|trans({}, 'Emails.Body', locale) }}
```
As @okom3pom said, this email [is for the shop, not the customer](https://github.com/PrestaShop/PrestaShop/blob/develop/controllers/front/OrderDetailController.php#L93).
```suggestion
      {{ 'Hi,'|trans({}, 'Emails.Body', locale) }}
```

sowbiba reviewed (commented):
LGTM
Waiting to resolve @atomiix and @okom3pom feedback to validate

matthieu-rolland reviewed (commented):
LGTM for me too once @atomiix 's feedback has been dealt with.

atomiix commented on .gitignore:
```diff
@@ -46,3 +46,5 @@
 # Embedded web-server pid file
 /.web-server-pid
+
+.idea
```
Not sure about that, I don't think we should be IDE files in the .gitignore

Progi1984 replied:
I approve you comment

Progi1984 commented on .gitignore:
```diff
@@ -46,3 +46,5 @@
 
 # Embedded web-server pid file
 /.web-server-pid
+
+.idea
```
```suggestion
```

Progi1984 reviewed (commented):
Some feedbacks

jolelievre commented:
@Progi1984 @atomiix You're both right, I was just lazy updating my global config 😅
It's done now and I removed the commit from the PR

sarahdib commented:
@jolelievre this one can be tested by a dev :)

atomiix commented:
Thanks @jolelievre
